### PR TITLE
Starting to use raywenderlich/swift-style-guide

### PR DESCRIPTION
**Pull request by rubenglobant**
started to work with swift style guide

### CODE REVIEW

**Review by rubenglobant — COMMENTED** [review]
starting point for swift style into this project

File: INActors/ActorsDescriptionViewController.swift [hunk 1]
```diff
@@ -54,11 +53,15 @@ class ActorsDescriptionViewController: UIViewController, UITableViewDelegate, UI
         
     }
     
-    public func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+}
+
+
```
Comment (jsamboni) [inline INActors/ActorsDescriptionViewController.swift]: This linebreak can be deleted

File: INActors/ActorsDescriptionViewController.swift [hunk 2]
```diff
@@ -69,5 +72,10 @@ class ActorsDescriptionViewController: UIViewController, UITableViewDelegate, UI
...
         return cell
     }
-
 }
+
+
+
+
+
+
```
Comment (jsamboni) [inline INActors/ActorsDescriptionViewController.swift]: All these linebreaks should be deleted

File: INActors/ActorsTableViewController.swift [hunk 3]
```diff
@@ -8,80 +8,84 @@
...
 var actorsInfo: [NSDictionary]?  // Array of dictionaries that save all data that come from the api
 
 // class defined in order to show the search results
-class ActorsTableViewController: UITableViewController, UISearchControllerDelegate, UISearchBarDelegate {
+class ActorsTableViewController: UIViewController {
 
     @IBOutlet weak var actorsSearchBar: UISearchBar!
+    @IBOutlet var tableView: UITableView!
+    
+
```
Comment (jsamboni) [inline INActors/ActorsTableViewController.swift]: extra linebreak can be deleted

File: INActors/ActorsTableViewController.swift [hunk 4]
```diff
@@ -8,80 +8,84 @@
...
     override func viewDidLoad() {
         super.viewDidLoad()
-        
-        self.actorsSearchBar.delegate = self
-        
     }
     
+}
+
+
```
Comment (jsamboni) [inline INActors/ActorsTableViewController.swift]: extra line break should be deleted

File: INActors/ActorsTableViewController.swift [hunk 5]
```diff
@@ -8,80 +8,84 @@
...
-    override func numberOfSections(in tableView: UITableView) -> Int {
-        return 1
+    // make a conexion with the api and shows the results in the table
+    func searchBar(_ searchBar: UISearchBar, textDidChange searchText: String) {
+        let requestActors: RequestTMDB = RequestTMDB()
+        requestActors.people(inTableViewActors: tableView, search: searchText)
+    }
+}
+
+
```
Comment (jsamboni) [inline INActors/ActorsTableViewController.swift]: extra line break should be deleted

File: INActors/ActorsTableViewController.swift [hunk 6]
```diff
@@ -130,7 +134,8 @@ class ActorsTableViewController: UITableViewController, UISearchControllerDelega
         
         // send data to ActorsDescriptionViewController
         self.navigationController?.pushViewController(description, animated: true)
-
+
```
Comment (jsamboni) [inline INActors/ActorsTableViewController.swift]: This line break can be deleted

File: INActors/ActorsTableViewController.swift [hunk 7]
```diff
@@ -8,80 +8,84 @@
...
     }
-
     
-    override func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
-
+    func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        
         let cell = tableView.dequeueReusableCell(withIdentifier: "actorsCell", for: indexPath) as! ActorsCellViewController
         actor = actorsInfo?[indexPath.row]
         name = actor?["name"] as? NSString
```
Comment (jsamboni) [inline INActors/ActorsTableViewController.swift]: i believe we can use String here instead of NSString

**Review by jsamboni — CHANGES_REQUESTED** [review]
A couple of changes to be made before merging but overall looks nice!

**Review by jsamboni — APPROVED** [review]
looks much better!

**Comment by rubenglobant** [comment]
changes merged

**Pull request merged**